### PR TITLE
CRM457-983: Add more validations and formatting

### PR DESCRIPTION
**Pull request by patrick-laa**
## Description of change
Following first round of QA, add some missed error messages, and ensure gbp values are formatted to 2 decimal places.

 [Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-983)

### CODE REVIEW

File: config/locales/en/prior_authority/alternative_quotes.yml [hunk 1]
```diff
@@ -97,9 +97,13 @@ en:
...
             travel_cost_per_hour:
               greater_than: The cost per hour must be more than 0
               not_a_number: The cost per hour must be a number, like 25
+              blank: To add travel costs you must enter both the time and the hourly cost
+            additional_cost_list:
+              blank: To add additional costs you must both a list of the additional costs and the total cost
             additional_cost_total:
               not_a_number: The cost must be a number, like 25
-
+              blank: To add additional costs you must both a list of the additional costs and the total cost
```
Comment (jsugarman) [inline config/locales/en/prior_authority/alternative_quotes.yml:109]: This english looks like it is missing something, perhaps...

```suggestion
              blank: To add additional costs you must both add a list of the additional costs and the total cost
```

File: spec/forms/prior_authority/steps/alternative_quotes/detail_form_spec.rb [hunk 2]
```diff
@@ -60,6 +67,36 @@
...
+      let(:additional_cost_list) { 'Photocopying' }
+
+      it 'returns false' do
+        expect(subject.save).to be false
+      end
+
+      it 'adds an appropriate error message' do
+        subject.save
+        expect(subject.errors[:additional_cost_total]).to include(
+          'To add additional costs you must both a list of the additional costs and the total cost'
```
Comment (jsugarman) [inline spec/forms/prior_authority/steps/alternative_quotes/detail_form_spec.rb:96]: ditto here

**Review by jsugarman — APPROVED** [review]
Just some missing english in an error message

File: config/locales/en/prior_authority/alternative_quotes.yml (hunk 1 above)
Comment (patrick-laa) [reply config/locales/en/prior_authority/alternative_quotes.yml]: Good spots, thanks!

**Pull request merged**